fe/src/theme.ts

```diff
@@ -0,0 +1,21 @@
+import { extendTheme } from '@chakra-ui/react';
+
+// Let's say you want to add custom colors
```

Comment:
Just a nitpick, but comments like these don't add anything

Reply:
I mean it adds the knowledge that you can add custom colors like this. I don't say its super valuable, but it makes it more clear imho. And its not actively hurting anybody. @tomimarkus991 your choice.

Reply:
Okay, i guess so. Doesn't hurt anyone.